docs/en/observability/whats-new.asciidoc

```diff
@@ -3,21 +3,272 @@
+having a rich tool that enables this type of data visualization and manipulation is a critical part of any analytics solution.
+{kib} currently has our Lens product which is an incredibly powerful tool, but given that it sits on top of the Elastic Stack,
+it needs to be able to cater to all kinds of different data and requires the users to be familiar with things like index patterns and field names.
+This meant that there was often a barrier to most users who weren't familiar with these
+things that prevented them from taking advantage of the power of Lens.
+We wanted to build a more focussed experience for Observability users that was based on the
+power of Lens but was abstracted away from the complexity that comes with a one-size-fits-all solution.
+
+[role="screenshot"]
+image::images/7.13-kpi-over-time.png[new exploratory view]
+
+We're incredibly excited to launch the technical preview of the new Observability Exploratory View in 7.13.
+With this release, we are focussing on User Experience (RUM) and Uptime (Synthetics) data.
+Users will be able to chart time series and distributions of key measurements and then filter and
+break down the data in a completely familiar way.
+As we evolve this new feature, expect to see more data types coming in, the ability to compare different user cohorts across multiple series,
+and comparisons across different periods of time.
+Any chart created in the Exploratory view can be opened in Lens for further editing or even embedded across {kib}
+anywhere existing visualizations can be embedded today.
+
+[role="screenshot"]
+image::images/7.13-kpi-over-time-2.png[new exploratory view]
+
+To access the Exploratory View, open Uptime or User Experience and click on the "Analyze data" link in the upper right-hand corner of the screen.
+
+See {observability-guide}/exploratory-data-visualizations.html[Exploratory data visualizations] for more information.
+
+[discrete]
+== Improved infrastructure monitoring widget on Observability Overview
+
+We updated the density of information about monitored infrastructure on the Observability Overview page,
+improving the ability to quickly find causes of problems and troubleshooting the root cause.
+The new Metrics widget now shows top hosts in your infrastructure with the largest resource footprint.
+
+[role="screenshot"]
+image::images/7.13-improved-infra-mon.png[Improved infrastructure monitoring widget]
+
+[discrete]
+== Supporting runtime fields stored in {kib} index patterns in Logs UI
+
+{kib} applications such as {kib} index pattern editor, Lens, and Discover, allow the creation of runtime fields. The Logs UI now extends support for choosing {kib} patterns and displaying and querying runtime fields in Logs UI.
+
+You can use {kib} index patterns to power the Logs UI, instead of choosing {es} index name patterns. This simplifies setup and administrative overhead and enables the visualization and querying of runtime fields stored in {kib} index patterns.
```

Comment:
I haven't kept up with the Kibana changes, but I'mnot sure what the difference is between a Kibana index pattern and an Elasticsearch index name pattern. :-(

Reply:
@mukeshelastic can you comment on this? I'm not finding anything in the Kibana runtimes fields docs that differentiates between an ES / Kib index pattern.